locations.xml

```diff
@@ -156264,6 +156264,7 @@
       <LocationId>badajoz</LocationId>
     </FallbackLocations>
     <Names>
+      <Name language="Arabic" value="Baṭalyūs" comment="Based on the name of Taifa(?)" />
```

Comment:
This should be placed under badajoz. This location (lusitania) already inherits names from badajoz when its own are not available - which is just the case for Arabic.

Also, badajoz already has an Arabic name "Baṭalyaws", you can just update that one instead, as yours has the correct diacritics.

Reply:
BTW: The locations from which the names are inherited are defined under `<FallbackLocations></FallbackLocations>`